source/fundamentals/indexes.txt

```diff
@@ -56,9 +56,9 @@ Operational Considerations
 
 To improve query performance, build indexes on fields that appear often in
 your application's queries and operations that return sorted results.
-Each index that you add consumes disk space and memory when active so you
-should track index memory and disk usage for capacity planning. In addition,
-when a write operation updates an indexed field, MongoDB also has to update
+Each index that you add consumes disk space and memory when active, so you
+may want to track index memory and disk usage for capacity planning. In addition,
+when a write operation updates an indexed field, MongoDB also updates
 the related index.
```

Comment:
```suggestion
Each index that you add consumes disk space and memory when active, so it might be necessary to
track index memory and disk usage for capacity planning. In addition,
when a write operation updates an indexed field, MongoDB also updates
the related index.
```